api/src/pcapi/alembic/versions/20240216T134646_6d2971ccc616_remove_favorite_mediationid_foreign_key.py

```diff
@@ -0,0 +1,21 @@
+"""
+remove favorite mediationId foreign_key
```

Comment:
Je micro-chipote, mais cette migration ne retire pas la contrainte de clé étrangère. Elle retire carrément la colonne.

```suggestion
Remove favorite.mediationId column
```

Reply:
`Remove favorite.mediationId column`
FAIT